server/controllers/auth.js

```diff
@@ -32,11 +32,11 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
   //creating the email and adding the id to the profile
   const profile = await Profile.create({
     email: user.email,
-    _id: user._id
-  })
+    _id: user._id,
+  });
 
-  if(profile){
-    console.log("Profile created")
+  if (profile) {
+    console.log("Profile created");
   }
```

Comment:
This doesn't even do anything other than log a message so it could be removed from the back-end. Thanks!

Reply:
This was not part of my implementation but I have removed it.